Cargo.toml

```diff
@@ -7,3 +7,9 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+secp256k1 = { version = "0.20.2", features = ["rand"] }
```

Comment:
Would be nice top use `^` here as a default:  
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements

Reply:
yes